arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/UberjarSimpleContainer.java

```diff
@@ -195,7 +195,7 @@ public void start(Archive<?> archive) throws Exception {
         // project dependencies
         FileSystemLayout fsLayout = FileSystemLayout.create();
         DeclaredDependencies declaredDependencies =
-                DependencyDeclarationFactory.newInstance(fsLayout).create(resolvingHelper);
+                DependencyDeclarationFactory.newInstance(fsLayout).create(fsLayout, resolvingHelper);
```

Comment:
This is a bit ugly. It's because, having injected `DependencyDeclarationFactory` via SPI, I can no longer pass fsLayout as a constructor parameter.
An alternative I could think of is to introduce another layer of factory/provider but I think it would make the code even worse.